app/views/share/_embed_meta.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= ts("Chapters: ") %><%= work.chapter_total_display %><br />
```

Comment:
It's counterintuitive, but iirc, we actually _want_ the entire share code smooshed onto one line like it is on production. If it's all pretty and readable on multiple lines like this, Dreamwidth and Livejournal treat each line like a paragraph, e.g.
```
Title (X words) by name

Chapters: 1/1

Fandom: Testing
```
instead of 
```
Title (X words) by name
Chapters: 1/1
Fandom: Testing
```